services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/tenant/AutowiredTenantInformationService.java

```diff
@@ -32,6 +37,7 @@
 public class AutowiredTenantInformationService implements TenantInformationService {
 
     private static final Logger LOG = LoggerFactory.getLogger(AutowiredTenantInformationService.class);
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
why do we need this?

Reply:
See: https://github.com/eclipse/hono/pull/2094#discussion_r471465046